config/cluster-rbac/auth_proxy_client_clusterrole.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
maybe put this into `cluster-install/rbac`?

Reply:
moved cluster-install into quick-cluster-install